tests/utils/runJS.test.tsx

```diff
@@ -107,11 +106,10 @@ test(
   }),
 );
 
-test(
+test.skip(
```

Comment:
Double-checking you meant to leave this in. 😉

Reply:
Yeah this fails because the source map code is commented out because the WMR based module loader doesn't support source maps. I'm opening a PR next week probably with adding the source map support back